app/src/main/java/org/sopt/pingle/data/model/remote/response/ResponseNewGroupKeywordsDto.kt

```diff
@@ -0,0 +1,18 @@
+package org.sopt.pingle.data.model.remote.response
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.sopt.pingle.domain.model.NewGroupKeywordEntity
+
+@Serializable
+data class ResponseNewGroupKeywordsDto(
```

Comment:
```suggestion
data class ResponseNewGroupKeywordDto(
```
어차피 리스트로 받으니까 이런 네이밍은 어떨까요 ㅋㅋ ㅠ